### PR TITLE
Adds a few xrefs to IR docs and adds an additional note to inform use…

### PR DESCRIPTION
**Pull request by stevsmit**
…rs to configure storage

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-35907

Link to docs preview:
Additional resources:
Line change: https://78120--ocpdocs-pr.netlify.app/openshift-enterprise/latest/registry/configuring-registry-operator#registry-removed_configuring-registry-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xiuwang** [comment]
LGTM

File: modules/registry-removed.adoc [hunk 1]
```diff
@@ -20,4 +20,4 @@
 
 On platforms that do not provide shareable object storage, the OpenShift Image Registry Operator bootstraps itself as `Removed`. This allows `openshift-installer` to complete installations on these platform types.
 
-After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`.
+After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`. Then, you must configure storage.
```
Comment (GroceryBoyJr) [inline modules/registry-removed.adoc:23]:
```suggestion
After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`.  When this is completed, then you must configure storage.
```

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM, only one suggestion.

File: modules/registry-removed.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply modules/registry-removed.adoc:23]: @stevsmit I don't think there is style guidance on this but I do believe it translates a little better. Your call man.

**Pull request merged**